### PR TITLE
Add Firefox Artefact

### PR DESCRIPTION
**Pull request by mikaelkrief**
Add Firefox artefact

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@mikaelkrief__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by mikaelkrief** [comment]
Yes, admins can verify this patch

**Comment by kbdavis07** [comment]
Firefox would be nice to have.

Please approve soon :smile:

File: Artifacts/windows-firefox/Artifactfile.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "title": "Firefox",
+  "description": "Installs Firefox using the Chocolatey package manager",
+  "tags": [
+    "Windows",
+    "Firefox"
+  ],
+  "iconUri": "https://chocolatey.org/content/packageimages/Firefox.44.0.2.png",
+  "targetOsType": "Windows",
+   "parameters": {
+        "installUsername": {
```
Comment (mithunshanbhag) [inline Artifacts/windows-firefox/Artifactfile.json]: @mikaelkrief : I see that this artifact requires the user to pass in a username and password (for installing firefox under a specific user's credentials). Would it be possible to eliminate the need for these parameters?

The general pattern that most artifacts follow is to create a temporary admin account to install the chocolatey package (Example: https://github.com/Azure/azure-devtestlab/tree/master/Artifacts/windows-7zip). This way, the user is not required to pass in a user name and password. 

Will that work for this artifact?

**Comment by mikaelkrief** [comment]
@mithunshanbhag : Ok I'll try with temporary user, like 7-zip

**Comment by mikaelkrief** [comment]
use temporary user for install is commited #62

**Comment by mithunshanbhag** [comment]
LGTM. Merging this to master.

**Pull request merged**